config/utils.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func getConfigYaml(path string) ([]byte, error) {
+	if strings.HasPrefix(path, "http") {
+		res, err := http.Get(path)
+		if err != nil {
+			return []byte{}, err
+		}
+
+		if res.StatusCode != 200 {
+			return []byte{}, errors.New(fmt.Sprintf("Request returned: %d", res.StatusCode))
+		}
+
+		resBody, err := ioutil.ReadAll(res.Body)
```

Comment:
ioutil is deprecated see [here](https://pkg.go.dev/io/ioutil#ReadAll)

You shuold use 
```suggestion
		resBody, err := io.ReadAll(res.Body)
```